account_payment_order_validation_move_line/wizard/account_payment_line_create.py

```diff
@@ -0,0 +1,24 @@
+# © 2015-2016 Akretion (<https://www.akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountPaymentLineCreate(models.TransientModel):
+    _inherit = "account.payment.line.create"
+
+    select_for_payment_move_line = fields.Boolean(
+        string="Take only to pay",
+        related="order_id.payment_mode_id.default_selected_for_payment_id",
```

Comment:
This field should not be related.
beeing related with readonly=False, if the user changes it on the wizard, it will also change on the payment mode, changing the default for next time!

It should be a regular field, which can be filled in the default.